### PR TITLE
fix speech bot crashing issue [done]

### PR DESCRIPTION
**Pull request by MSFTserver**
fixed crashing issue
added to work in channels: #speech, #design,  #meme, #random, #random-nsfw, #bot-sandbox

### CODE REVIEW

File: bot/modules/speech.js [hunk 1]
```diff
@@ -18,7 +18,12 @@ exports.speech = {
 	description: "gets top claim from spee.ch, coming soon post to spee.ch",
 	process: function(bot,msg,suffix){
 		
-		var ChannelID = "363085078403874823"
+		var ChannelID1 = "363085078403874823"
```
Comment (filipnyquist) [inline bot/modules/speech.js]: These should be moved to an array inside of the config called speech with the channels as an array.

File: bot/modules/speech.js [hunk 2]
```diff
@@ -243,7 +244,7 @@ request.post(
 };
 
 function inPrivateOrBotSandbox(msg){
-  if((msg.channel.type == 'dm') || (msg.channel.id === ChannelID)){
+  if((msg.channel.type == 'dm') || (msg.channel.id === ChannelID1) || (msg.channel.id === ChannelID2) || (msg.channel.id === ChannelID3) || (msg.channel.id === ChannelID4) || (msg.channel.id === ChannelID5) || (msg.channel.id === ChannelID6)){
```
Comment (filipnyquist) [inline bot/modules/speech.js]: use the config array and then array.includes(msg.channel.id) instead  of all of these or statements :)

File: bot/modules/speech.js [hunk 3]
```diff
@@ -17,17 +18,13 @@ exports.speech = {
...
 	process: function(bot,msg,suffix){
-		
-		var ChannelID = "363085078403874823"
-		
-		var command = "!speech"
-	words = suffix.trim().split(' ').filter( function(n){return n !== "";} );
-          var imagename = words[0];
-  
-  	  if(!inPrivateOrBotSandbox(msg)){
+		if(hasSpeechChannels(msg) === false){
```
Comment (filipnyquist) [inline bot/modules/speech.js]: Could just do !hasSpeechChannels(msg) instead of === false

File: config/default.json.example [hunk 4]
```diff
@@ -17,4 +17,7 @@
   "perms": ["LBRY MODS","LBRY TEAM"], // Roles that have access to all commands.
   "logchannel": "371620338263523328" // Channel to log the bots moderation..
   }
```
Comment (filipnyquist) [inline config/default.json.example]: missing a , before speechbot

**Comment by MSFTserver** [comment]
this should be good now

**Pull request merged**